### DISCUSSION TITLE
Failed to make invite request: Error inviting user

### DISCUSSION THREAD

**Discussion by YordanPZ** [supabase/supabase#20408] [Questions]
I’m trying to add a new user to my project in the Auth table, but I’m encountering the error mentioned in the title. Additionally, I’m unable to perform a password change for already created users, receiving this error: {code: 500, msg: "Unable to process request", error_id: "8450964940d108d9-EZE"}. In cases where an email does manage to get sent, it never arrives; I’m not receiving any type of email. Approximately two days ago, everything was working perfectly I could do all the aforementioned tasks without any issues.

**Comment by GaryAustin1** [comment]
You should check your Postgres and auth logs for errors.
